portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/dependencies/com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration~Auth0.config

```diff
@@ -4,8 +4,8 @@ discoveryEndPointCacheInMillis=L"360000"
 idTokenSigningAlgValues=["RS256"]
 issuerURL=""
 jwksURI=""
-openIdConnectClientId="6Ko608nCF44eCNWfZlQk2IjQZXm8k3cG"
-openIdConnectClientSecret="Fetz2IxF5yECfBGlhGQJj5h4EWXQ6taXxqxkB3uT1vG3dRFr-nHGF3GzbAWfIrUk"
+openIdConnectClientId="${openid.provider.auth0.client.id}"
+openIdConnectClientSecret="${openid.provider.auth0.client.secret}"
```

Comment:
Should be openid.provider.auth0.client.password: https://github.com/liferay/liferay-jenkins-ee/blob/master/commands/build.properties#L1093